src/Symfony/Bundle/SecurityBundle/Debug/TraceableFirewallListener.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\SecurityBundle\Debug;
+
+use Symfony\Bundle\SecurityBundle\EventListener\FirewallListener;
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+
+/**
+ * Firewall collecting called listeners.
+ *
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ *
+ * @final since version 3.4
```

Comment:
To ease maintenance while keeping extensibility: one could add more info per listener and if a day we don't need this method for dumping the object, the breaking change would be free